test cases/common/152 index customtarget/meson.build

```diff
@@ -56,6 +56,13 @@ custom_target(
   input: gen[0],
   output: 'foo',
   command: [find_program('check_args.py'), '@INPUT@'],
+  build_by_default: true,
+)
+
+custom_target(
+  output: 'foo2',
+  command: [find_program('check_args.py'), gen[0]],
+  build_by_default: true,
```

Comment:
This changes are cause this failure: https://dev.azure.com/jussi0947/jussi/_build/results?buildId=18382&view=logs&j=447df341-d5a1-5d0e-9651-c9f66402368a&t=12f22127-2cd1-5ea8-0ccb-9ba120a59cea&l=2940

Reply:
Oh god, digging that one I found yet another inconsistency: when using `@INPUT@`, `gen[0]` is passed as path relative to builddir. But when passing `gen[0]` directly on the command kwarg, it's passed as absolute path. That test was supposed to catch that but since it didn,t have `build_by_default: true,` it never actually tested it...